### PR TITLE
chore: update `.npmignore` to exclude misc files

### PR DESCRIPTION
**Pull request by G-Rath**
The latest release (10.1.3) included a few misc files:

```
❯ npx npm@7 diff --diff json-schema-to-typescript@10.1.2 --diff json-schema-to-typescript@10.1.3 --diff-name-only
npx: installed 254 in 13.455s
dist/src/cli.js
package.json
yarn-error.log
package-lock.json
```

### CODE REVIEW

File: .npmignore [hunk 1]
```diff
@@ -11,3 +12,5 @@ circle.yml
 ARCHITECTURE.md
 CONTRIBUTING.md
 .editorconfig
+package-lock.json
+yarn.lock
```
Comment (bcherny) [inline .npmignore:16]: This should be checked in.

**Review by bcherny — CHANGES_REQUESTED** [review]
LG! One minor request.

File: .npmignore (hunk 1 above)
Comment (G-Rath) [reply .npmignore:16]: Why? This is used by `npm` when publishing, not for `git`.

Lock files like `yarn.lock` do nothing in published packages, so shouldn't be included.

Comment (bcherny) [reply .npmignore:16]: Ah, you're right. Good catch!

**Review by bcherny — APPROVED** [review]
LG!

**Pull request merged**